aptos-move/move-examples/fungible_asset/sources/managed_fungible_asset.move

```diff
@@ -0,0 +1,190 @@
+/// This module provides an addtional ready-to-use solution on top of `FungibleAssetMetadata` that manages the refs of
+/// mint, burn and transfer for the creator in a straightforward scheme. It offers creators to destory any refs in an
+/// on-demand manner too.
+module fungible_asset::managed_fungible_metadata {
+    use aptos_framework::fungible_asset::{Self, MintRef, TransferRef, BurnRef, FungibleAsset};
+    use aptos_framework::object::{Self, Object, ConstructorRef};
+    use std::error;
+    use std::signer;
+    use std::string::String;
+
+    /// MintRef existence error.
+    const EMINT_REF: u64 = 1;
+    /// TransferRef existence error.
+    const ETRANSFER_REF: u64 = 2;
+    /// BurnRef existence error.
+    const EBURN_REF: u64 = 3;
+    /// Not the owner.
+    const ENOT_OWNER: u64 = 4;
+    /// Refs existence errors.
+    const EMANAGED_FUNGIBLE_ASSET_REFS: u64 = 5;
+
+    #[resource_group_member(group = aptos_framework::object::ObjectGroup)]
+    /// Hold refs to control the minting, transfer and burning of fungible assets.
+    struct ManagedFungibleAsset has key {
+        mint_ref: MintRef,
+        transfer_ref: TransferRef,
+        burn_ref: BurnRef,
```

Comment:
bend you knees to david? what if a creator wants to show that they doesn't have the ability to freeze account?

Reply:
Rofl... Managed isn't intended for production use cases. I think real assets likely have their own framework.

Reply:
> Rofl... Managed isn't intended for production use cases. I think real assets likely have their own framework.

I think you will agree that most ppl just copy&publish the code... As we agreed 99% are not likely to build their own so the default example should be a little bit more comprehensive.

Reply:
Lol there's a point about keeping the examples simple as the simpler they are, the easier it'd be for devs to understand and customize. Furthermore, I'm thinking we can add more different examples, similar to what OpenZeppelin has.

Reply:
If we have more examples later on, I am cool with it.

Reply:
Both of us will :)